msupply-foundation/open-msupply
# Implement name soft deletes

Fixes #3555

# 👩🏻‍💻 What does this PR do?
When receiving a name MERGE record, the deleted name gets soft-delete now.

# 🧪 Testing

See issue, but found this easier:
- add a dispensary store to the central server site
- create name "a1" and "a2" on the central server
- sync and see names on the remote sites
- merge "a1" and "a2"
- sync, there should only be one name visible on the remote site now
